mux/node.go

```diff
@@ -84,18 +84,16 @@ type staticNode struct {
 func (n *staticNode) Match(path string) (Node, middleware.Middleware, context.Params, string) {
 	nameLength := len(n.name)
 	pathLength := len(path)
-
-	if pathLength >= nameLength && n.name == path[:nameLength] {
+	if pathLength >= nameLength && n.name == path[:nameLength] || regexp.MustCompile(`{|}`).MatchString(n.name) {
 		if nameLength+1 >= pathLength {
-			return n, n.middleware, make(context.Params, n.maxParamsSize), ""
+			// is there a better solution here ? it stopped working once the braces were included in node.name
+			return n, n.middleware, context.Params{{Key: "param", Value: path}}, ""
```

Comment:
`make(context.Params, n.maxParamsSize)` params were initialised with size `n.maxParamsSize` on purpose
to reduce time complexity and not to merge one or more params together if we know the max size of it